docs/gen-1/red-blue/main-glitchless/intermediate-route/README.md

```diff
@@ -0,0 +1,714 @@
+
+### Glossary:
+- Moves: HA = Horn Attack, MP = Mega Punch, BB = Bubble Beam, WG = Water Gun, TB = Thunder Bolt, Blizz = Blizzard, PS = Poison Sting, HD = Horn Drill, RS = Rock Slide
+  - (MOVE NAME): A situational move used to finish off a pokemon when a damage range is missed
+- IT = Instant Text
+
+### Before you Start:
+- Clear any existing save file by pressing Up + B + Select on the game title screen
+- Set your options to fast text and animations OFF
+
+## NIDORAN SPLIT
+
+Rival 1:
+- Tail Whip x1 + Tackle Spam
+> Check stats on level up after rival 1
++ 10 ATK - Tail Whip x3 on Weedle Guy in forest
++ 12 ATK - Tail Whip + Tacklex2 on L3 Rat encounter only
++ 12 DEF is bad defense, expect Geodude & Onix to do more damage (see Defensive Ranges)
++ 11 SPC is likely bad special (3 hit on Geodude and/or Onix)
++ 12 SPC is guarenteed good special (2 hit on Geodude & Onix)
+
+Route 1
+- spam tackle to KO one encounter that is L2 or L3
+
+Viridian Mart
+- Buy 3, 4 or 5 Poke Balls depending on your yolo ball consistency
+- note: if buying 5 balls, buy one fewer potion later
+
+Do [Nido Manip](../resources/nido-manip.md)
+> Optional: Do [Triple Extended Manip](../resources/triple-extended)
+
+## BROCK SPLIT
+
+- Pick up the hidden Tree Potion if not doing triple extended
+
+- Pick up the Antidote
+
+> If Squirtle is under 16 hp, pick up the hidden Weedle Guy Potion and heal before the next fight.
+> Always pick up the Weedle Guy Potion if you bought 5 balls or skipped tree potion on good pace.
+
+Weedle Guy:
+- Tail Whip x2 (Tail Whip a third time with 10 ATK)
+- Tackle Spam [Heal if HP < 5]
+
+Swap Squirtle and Nido at some point
+- Toss antidote if you are healing Squirtle before Brock
+- Potion if HP is below 11
+
+Pewter Mart
+- Buy 8 Potions (7 if you bought 5 balls)
+
+Brock:
+- Swap to Squirtle
+- Bubble x2-3 on Geodude [Heal if HP < 8]
+- Switch to Nido, then swap back to Squirtle
+- Bubble x2-3 on Onix [Heal if HP < 7]
+
+## ROUTE 3 SPLIT
+
+Change Options to Set before leaving the Gym
+
+BC1:
+- Leer, HA x2(Tackle)
+- Leer, HA + (HA or Tackle if you can)
+- Leer, HA + Tackle
+
+- Menu: Use a potion if HP < 25 & 
+*Toss antidote if healing before Shorts Guy*
+> with 25-27 hp use HAx2 (+Tackle) on Rat
+
+Shorts Guy:
+- Leer, HA x2  [Heal if HP < 10, @-1 if HP < 16]
+- Leer, HA x2  [Heal if HP < 4, @-1 hp < 5, @-2 if hp < 6]
+> Potion turn 1 of wrap if you don't live a 4 or 5-turn-wrap.
+
+BC2:
+- HA + HA or Tackle
+- HA x3 (+ HA or Tackle)
+- HA x2 (Tackle turn 2 if Weedle hit 2 string shots) (Heal if hp < 4)
+- HA x2 + Tackle or finish with HA if Metapod outspeeds you
+
+- if hp < 4 Menu before the next fight and potionx2
+
+BC3:
+- HA x2
+- HA x3-4(finish with Tackle if you can)
+
++ Catch a Flyer
+  + Tackle then toss a ball at any flyer except L8 Pidgey where you HA then toss a ball
+  > Exception: if only 2 HAs are left, then weaken L8 Pidgey with Leer + Tackle
+
+## MT. MOON SPLIT
+
+**Moon Manip Save**
+- 1- 8
```

Comment:
Thoughts on simplifying this to "heal to 36+"

Reply:
Yes, did this and added an if confident with yolo ball under 9 hp then delay heal clause.